v-next/hardhat/src/internal/builtin-plugins/network-manager/request-handlers/handlers/accounts/local-accounts.ts

```diff
@@ -276,15 +276,22 @@ export class LocalAccountsHandler extends ChainId implements RequestHandler {
       };
     });
 
-    // TODO: Fix after the alpha release
```

Comment:
Can this code be already deleted? Tests are passing so I assume so

Reply:
I'm not sure i understood this comment. Haven't you replaced this code in this same pr?

Reply:
Yes, let me rephrase: I wanna double check that is ok to already remove this code. Or if it should stay for now. In that case I would have to skip the tests for the moment

Reply:
Oh, I see! No, the new version without this assert is correct. I just added it as a quick fix during the retreat.